src/objects/_main-wrapper.scss

```diff
@@ -2,52 +2,27 @@
 @import "../tools/all";
 @import "../helpers/all";
 
-// Example usage with Breadcrumbs, phase banners, back links:
-// <div class="govuk-width-container">
-//   <!-- Breadcrumbs, phase banners, back links are placed in here. -->
-//   <div class="govuk-main-wrapper">
-//       <!-- Wrapper for the main content of your page which applies padding
-//            to the top / bottom -->
-//   </div>
-// </div>
-//
-// Example usage without Breadcrumbs, phase banners, back links:
-// <div class="govuk-width-container">
-//   <div class="govuk-main-wrapper govuk-main-wrapper--l">
-//       <!-- Wrapper for the main content of your page which applies padding
-//            to the top / bottom -->
-//   </div>
-// </div>
-
-
-@mixin govuk-main-wrapper {
```

Comment:
Maybe I'm being too trigger happy and we should just leave these, I'm just not sure why they're part of the public api.

Reply:
Yeah, I'm not sure either…

By the looks of it, I added the mixins in https://github.com/alphagov/govuk-frontend/commit/4c01202842352f6f8488065cab4d9dd1db976b1c but I've no idea why.